ZPastel.API/Resources/OrderResource.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ZPastel.API.Resources
+{
+    public class OrderResource
+    {
+        public long Id { get; set; }
+        public decimal TotalPrice { get; set; }
+        public IList<OrderItemResource> OrderItems { get; set; } = new List<OrderItemResource>();
+        public string CreatedByUsername { get; set; }
+        public long CreatedById { get; set; }
+        public DateTime CreatedOn { get; set; }
```

Comment:
Can this be nullable?

Reply:
Changed CreatedBy and LastModifiedBy from OrderResource, OrderItemResource and PastelResource to be nullable